bundle/manifests/rhods-operator.clusterserviceversion.yaml

```diff
@@ -154,22 +148,36 @@ metadata:
             "ray": {
               "managementState": "Managed"
             },
-            "workbenches": {
-              "managementState": "Managed"
-            },
             "trainingoperator": {
               "managementState": "Removed"
             },
             "trustyai": {
               "managementState": "Removed"
+            },
+            "workbenches": {
+              "managementState": "Managed"
             }
           }
         }
       }
-    operators.operatorframework.io/builder: operator-sdk-v1.31.0
+    operators.operatorframework.io/builder: operator-sdk-v1.24.1
+    operators.operatorframework.io/internal-objects: '[dscinitialization.opendatahub.io]'
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
     repository: https://github.com/red-hat-data-services/rhods-operator
-  name: rhods-operator.v2.13.1
+    features.operators.openshift.io/disconnected: "true"
+    features.operators.openshift.io/fips-compliant: "false"
+    features.operators.openshift.io/proxy-aware: "false"
+    features.operators.openshift.io/tls-profiles: "false"
+    features.operators.openshift.io/token-auth-aws: "false"
+    features.operators.openshift.io/token-auth-azure: "false"
+    features.operators.openshift.io/token-auth-gcp: "false"
+    description:
+      Operator for deployment and management of Red Hat OpenShift AI
+    support: Red Hat OpenShift AI
+    operators.openshift.io/valid-subscription: '["OpenShift AI"]'
+    operatorframework.io/suggested-namespace: redhat-ods-operator
+    operators.openshift.io/infrastructure-features: '["disconnected"]'
+  name: rhods-operator.2.13.1
```

Comment:
same here i think devops will replace exact version here,
can put a 2.13.2 instead ?

Reply:
That's right